runtime/polkadot/src/lib.rs

```diff
@@ -1598,8 +1598,9 @@ impl Get<&'static str> for StakingMigrationV11OldPallet {
 }
 
 pub type Migrations = (
+	pallet_balances::migration::ResetInactive<Runtime>,
 	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckAccount>,
```

Comment:
```suggestion
	// We need to apply this migration again, because `ResetInactive` resets the state again.
	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckAccount>,
```